tools/scripts/brokenLinkChecker/brokenLinkChecker.json

```diff
@@ -18,32 +17,15 @@
     "http://www.boston.com/lifestyle/health/blogs/daily-dose/2013/12/12/will-teaching-kids-code-boost-math-sat-scores/nVHKnP9aCbQ2ejUcoTKAQL/blog.html",
     "http://www.bostonglobe.com/business/2013/11/26/computer-science-course-breaks-stereotypes-and-fills-halls-harvard/7XAXko7O392DiO1nAhp7dL/story.html?s_campaign=sm_tw",
     "http://www.bostonglobe.com/metro/regionals/west/2013/12/29/more-schools-try-crack-code-computer-programming/6E7r6lhgwDL6ugGVKWe7xO/story.html",
+    "http://www.coloradotechnology.org/",
     "http://www.csta.acm.org/",
     "http://www.csteachers.org/",
     "http://www.expedia.com/",
     "http://www.fundacionsadosky.org.ar/",
     "http://www.girlswhocode.com/",
     "http://www.infosys.com/infosys-foundation/infosys-usa-foundation",
     "http://www.infosys.com/infosys-foundation/infosys-usa-foundation/Pages/index.aspx",
-    "http://www.linkedin.com/in/adrieltam",
-    "http://www.linkedin.com/in/amyhirotaka",
-    "http://www.linkedin.com/in/apone",
-    "http://www.linkedin.com/in/aray746",
-    "http://www.linkedin.com/in/aydinghajar",
-    "http://www.linkedin.com/in/davidsbailey",
-    "http://www.linkedin.com/in/deutschrebecca",
-    "http://www.linkedin.com/in/janeteperez",
-    "http://www.linkedin.com/in/jeffweiner08",
-    "http://www.linkedin.com/in/jennalgarcia",
-    "http://www.linkedin.com/in/lkline/",
-    "http://www.linkedin.com/in/melissadglenn",
-    "http://www.linkedin.com/in/misskatiek",
-    "http://www.linkedin.com/in/reidhoffman",
-    "http://www.linkedin.com/in/roxanneemadi",
-    "http://www.linkedin.com/in/tcparker",
-    "http://www.linkedin.com/pub/james-gwertzman/0/9/362/",
-    "http://www.linkedin.com/pub/owen-astrachan/2/395/9b5/",
-    "http://www.linkedin.com/pub/stephanie-chiang/13/59b/112",
+    "http://www.linkedin.com/",
```

Comment:
We might need to add https://wwwlinkedin.com as well?

Reply:
Yup! Added in line 64